internal/database/user_roles.go

```diff
@@ -128,7 +176,47 @@ func (r *userRoleStore) BulkCreateForUser(ctx context.Context, opts BulkCreateFo
 	}
 
 	q := sqlf.Sprintf(
-		userRoleCreateQueryFmtStr,
+		userRoleAssignQueryFmtStr,
+		sqlf.Join(userRoleInsertColumns, ", "),
+		sqlf.Join(urs, ", "),
+		sqlf.Join(userRoleColumns, ", "),
+	)
+
+	rows, err := r.Query(ctx, q)
+	if err != nil {
+		return nil, errors.Wrap(err, "error running query")
+	}
+	defer rows.Close()
+
+	var userRoles []*types.UserRole
+	for rows.Next() {
+		ur, err := scanUserRole(rows)
+		if err != nil {
+			return userRoles, err
+		}
+		userRoles = append(userRoles, ur)
+	}
+
+	return userRoles, nil
+}
```

Comment:
This can be turned into ~1 line if you do

```go
var scanUserRoles = basestore.NewSliceScanner(scanUserRole)
```
```suggestion
	return scanUserRolesr.Query(ctx, q)
}
```

See: https://sourcegraph.sourcegraph.com/search?q=context:sourcegraph+NewSliceScanner%28&patternType=standard&sm=0&groupBy=repo